cl/phase1/forkchoice/forkchoice_mock.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/ledgerwatch/erigon/cl/pool"
 	"github.com/ledgerwatch/erigon/cl/transition/impl/eth2"
 	"github.com/ledgerwatch/erigon/cl/validator/sync_contribution_pool"
+	syncPoolMock "github.com/ledgerwatch/erigon/cl/validator/sync_contribution_pool/mock_services"
```

Comment:
fyi Uber's go style guide suggests using all lowercase (no camel case and no snake case) for both package names & import aliases:
- https://github.com/uber-go/guide/blob/master/style.md#package-names
- https://rakyll.org/style-packages/

would rename `syncPoolMock` to `syncpoolmock` and `mock_aggregation` to `mockaggregation` or `aggregationmock`